evergreen.yml

```diff
@@ -514,15 +514,15 @@ functions:
         working_dir: src/genny
         script: |
           set -eu
-          versions=("master" "8.0" "7.3" "7.0" "6.0" "5.0")
+          versions=("master" "8.0" "7.0" "6.0" "5.0")
```

Comment:
We will not backport the taskgen changes to 7.3, so we don't need to generate files for it.